libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -563,6 +563,16 @@ singleton AP_Winch method release_length void float'skip_check
 singleton AP_Winch method set_desired_rate void float'skip_check
 singleton AP_Winch method get_rate_max float
 
+include AP_Parachute/AP_Parachute.h depends APM_BUILD_TYPE(APM_BUILD_ArduPlane)||APM_BUILD_COPTER_OR_HELI
+singleton AP_Parachute depends APM_BUILD_TYPE(APM_BUILD_ArduPlane)||APM_BUILD_COPTER_OR_HELI
```

Comment:
I think `HAL_PARACHUTE_ENABLED` should work. Unfortunately parachute is currently something that can be turned off at the vehicle level, so it is not possible for scripting to tell if it will be there in all cases.

We would have to remove that vehicle level ability and leave a compile time warning for anyone who had been using it to move to the new `HAL_PARACHUTE_ENABLED`

Reply:
Agree with that. Tried with it on the first version but I was getting a compilation error complaining that HAL_PARACHUTE_ENABLED is not defined. Any clues on how should I properly refer to it?

Reply:
Maybe `defined(HAL_PARACHUTE_ENABLED) & HAL_PARACHUTE_ENABLED`

Reply:
Changed it. Used && instead of & as otherwise both are evaluated and the compilation fails if HAL_PARACHUTE_ENABLED is not defined